app/upgrades.go

```diff
@@ -30,6 +30,13 @@ func (app UmeeApp) RegisterUpgradeHandlers(bool) {
 
 	app.registerUpgrade3_1to3_3(upgradeInfo)
 	app.registerUpgrade3_2to3_3(upgradeInfo)
+	app.registerUpgrade3_3to3_4(upgradeInfo)
+}
+
+// performs upgrade from v3.3 -> v3.4
+func (app *UmeeApp) registerUpgrade3_3to3_4(_ upgradetypes.Plan) {
+	const planName = "v3.3-v3.4"
```

Comment:
This will be a direct upgrade, so let's simplify the name as we did with v3.2 and v3.1

Reply:
```suggestion
	const planName = "v3.4"
```